main/unbound/unbound.initd

```diff
@@ -10,10 +7,14 @@ description="unbound is a Domain Name Server (DNS) that is used to resolve host
 description_configtest="Run syntax tests for configuration files only."
 description_reload="Kills all children and reloads the configuration."
 
+# Upper case variables are here only for backward compatibility.
+: ${cfgfile:=${UNBOUND_CONFFILE:-/etc/unbound/$RC_SVCNAME.conf}}
+: ${checkcfg:=${UNBOUND_CHECKCONF:-/usr/sbin/unbound-checkconf}}
 
-UNBOUND_BINARY=${UNBOUND_BINARY:-/usr/sbin/unbound}
-UNBOUND_CHECKCONF=${UNBOUND_CHECKCONF:-/usr/sbin/unbound-checkconf}
-UNBOUND_CONFFILE=${UNBOUND_CONFFILE:-/etc/unbound/${SVCNAME}.conf}
+command=/usr/sbin/unbound
+command_args="-d $command_args"
+pidfile=$("$checkcfg" -o pidfile "$cfgfile" 2>/dev/null)
```

Comment:
Once again, pidfile location should be hard-coded in runscript (as `/run/$RC_SVCNAME.pid`), not read from program’s configuration file.